legate/core/_legion/operation.py

```diff
@@ -1200,8 +1200,12 @@ def __init__(
         query = legion.legion_memory_query_create(machine)
         legion.legion_memory_query_only_kind(query, legion.SYSTEM_MEM)
         legion.legion_memory_query_local_address_space(query)
-        assert legion.legion_memory_query_count(query) > 0
+        sysmem_count = legion.legion_memory_query_count(query)
+        assert sysmem_count > 0
         mem = legion.legion_memory_query_first(query)
+        if sysmem_count > 1:
```

Comment:
@manopapad FYI, Realm recently added another system memory of size 0 for external allocations. So, there are more than one memory of kind `SYSTEM_MEM` in the default setting. (You can turn it off with `-ll:ext_sysmem 0`.)